src/rrbot_control/package.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <package format="2">
-  <name>box_robot</name>
+  <name>rrbot_control</name>
```

Comment:
let's decide the name of your robot and name all places accordingly. Why box_robot or rrbot , but the name of the repository says kuka_robot? Where is the truth?

Reply:
i have change the all package name and files name .i think now its looking fine